var/spack/repos/builtin/packages/r-a4/package.py

```diff
@@ -29,10 +29,11 @@ class RA4(RPackage):
     """Automated Affymetrix Array Analysis Umbrella Package."""
 
     homepage = "https://www.bioconductor.org/packages/a4/"
-    url      = "https://www.bioconductor.org/packages/release/bioc/src/contrib/a4_1.24.0.tar.gz"
+    url      = "https://git.bioconductor.org/packages/a4"
     list_url = homepage
-    version('1.24.0', 'dfa17ec5b1914300360ff11f43955fdd')
+    version('1.24.0', git='https://git.bioconductor.org/packages/a4', commit='79b5143652176787c85a0d587b3bbfad6b4a19f4')
```

Comment:
Why?

Reply:
Because the bioconductor webpage dosen't keep the old version for download, but their git does, so if I change the url to git, I don't need to update each time they update the version.

Reply:
ok